### ISSUE TITLE
Not loading one page documents

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#75]
```
I am having trouble loading one page documents.

I am suffering similar symptoms as indicated in this post:

http://groups.google.com/group/flexpaper/browse_thread/thread/e8bf688c243662e1

I am using Win XP SP#, Flex 1.3b & PDF2SWF v0.9 and I am unable to
open single page documents. I can successfully open multipage
documents (2 pages and greater).

I am using the command string for PDF2SWF that is recommended in the
forums.

I noticed in an earlier post that there were problems with single page
documents and I appear to be experiencing the same symptoms. Has this
been fixed in latest versions. Do you have any suggestions on
what I might be doing wrong?

I have attached a sample one page document that I am having problems with.
```

Original issue reported on code.google.com by `coals1...@gmail.com` on 20 May 2010 at 8:30

Attachments:
- [stn013_1958_flex.swf](https://storage.googleapis.com/google-code-attachments/flexpaper/issue-75/comment-0/stn013_1958_flex.swf)

**Comment by GoogleCodeExporter** [comment]
```
Thanks for your sample file. It probably works fine when not using progressive 
loading. I have found an issue 
with progressive loading and some versions of PDS2SWF + 1 page documents. It 
will be fixed ASAP and new build 
will be out very soon. Attaching a dev build for you to test
```

Original comment by `erik.eng...@gmail.com` on 20 May 2010 at 9:18

Attachments:
- [FlexPaperViewer.swf](https://storage.googleapis.com/google-code-attachments/flexpaper/issue-75/comment-1/FlexPaperViewer.swf)

**Comment by GoogleCodeExporter** [comment]
Original comment by `erik.eng...@gmail.com` on 20 May 2010 at 9:18
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Have tried the dev build and it doesn't appear to solve the problem. Anything 
else
that I can do to help diagnose problem?
```

Original comment by `coals1...@gmail.com` on 20 May 2010 at 8:52

**Comment by GoogleCodeExporter** [comment]
```
i tried your doc with the dev build and it works.. you sure your browser hasnt 
cashed 
the older version of flexpaper? you can trigger a refresh by adding an argument 
after 
FlexPaperViewer.swf as such:

FlexPaperViewer.swf?hello
```

Original comment by `erik.eng...@gmail.com` on 20 May 2010 at 8:54

**Comment by GoogleCodeExporter** [comment]
```
Yes, I have gone back and checked all and it appears to be loading ok now. 
Thanks for
looking at it for me.
```

Original comment by `coals1...@gmail.com` on 21 May 2010 at 7:51

**Comment by GoogleCodeExporter** [comment]
```
fixed in 1.3.0 rc1
```

Original comment by `erik.eng...@gmail.com` on 23 May 2010 at 7:44
- Changed state: **Fixed**
